CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - The [Conjur Ansible role](https://galaxy.ansible.com/cyberark/conjur-host-identity) has been
   migrated to this collection, where it will be maintained moving forward.
   [cyberark/ansible-conjur-host-identity#30](https://github.com/cyberark/ansible-conjur-host-identity/issues/30)
+- Add `file_mode` boolean option to store the secret as a temporary file in /dev/shm/ and return its path.
```

Comment:
This should also include a link to the issue it is trying to resolve.

Reply:
I think we also need to add usage of this in the README too

Reply:
This is a UX issue but `file_mode` isn't quite as descriptive as it should be to explain this feature. Any suggestions for a better one maybe?

Reply:
Added a link to the Cyberark Commons post. Regarding the usage, I'm not sure where to put it in README.
I renamed file_mode to as_file